src/Azure.Functions.Cli/Helpers/PythonHelpers.cs

```diff
@@ -13,7 +13,7 @@ namespace Azure.Functions.Cli.Helpers
 {
     public static class PythonHelpers
     {
-        private static readonly string[] _workerPackages = new[] { "azure-functions==1.0.0a4", "azure-functions-worker==1.0.0a4" };
+        private static readonly string[] _workerPackages = new[] { "azure-functions==1.0.0a5", "azure-functions-worker==1.0.0a6" };
```

Comment:
is `azure-functions` intentionally `0a5` not `0a6`?

Reply:
@pragnagopa can you comment on this?

Reply:
Yes. No changes in azure-functions from a5  but azure-functions-worker had few bug fixes then went in.